scripts/wait-for.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# poll the proxy endpoint until we get a 0 return code or 5mins have passed, in that case exit 1
+timeout_time=$(($(date +%s) + 300))
+
+while (( $(date +%s) <= timeout_time )); do
+  if curl -X GET 'http://localhost:6666/health'; then
+    exit 0
+  else
+    sleep 20
```

Comment:
sleep 2 is prob better? otherwise the ci takes minimum 20seconds given Id assume the first ping is not going to work because container is still being spun up.